lighthouse-cli/chrome-launcher.ts

```diff
@@ -53,6 +53,7 @@ class ChromeLauncher {
     const flags = [
       '--remote-debugging-port=9222',
       '--disable-extensions',
+      '--disable-translate',
```

Comment:
lets add this too: 

* `--disable-default-apps `  "Disables installation of default apps on first run. This is used during automated testing"